resource-managers/kubernetes/integration-tests/src/test/scala/org/apache/spark/deploy/k8s/integrationtest/ClientModeTestsSuite.scala

```diff
@@ -17,13 +17,13 @@
 package org.apache.spark.deploy.k8s.integrationtest
 
 import org.scalatest.concurrent.Eventually
-import scala.collection.JavaConverters._
 
 import org.apache.spark.deploy.k8s.integrationtest.KubernetesSuite.{k8sTestTag, INTERVAL, TIMEOUT}
+import org.scalatest.Tag
```

Comment:
out of curiosity, why add the tag?

Reply:
As I was just wanting to debug this specific failing test it was useful to run it on its own repeatedly as I investigated the issue.  I backed out other changes I made to enable this (like not cleaning up the tests driver pod) so I will back this out as well

Reply:
BTW you don't need tags just to run specific tests.

e.g. in maven you can use `-Dsuites="*.SuiteName part of test name to test"

To do matching on the test name. Similar in sbt (`testOnly *.SuiteName -- -z part of test name to test`).

Reply:
I used tags just because that's what the existing scripting did.  I thought there was probably a Maven property to do the same but using tags was fastest path for me.